src/som/compiler/MethodGenerationContext.java

```diff
@@ -29,31 +29,40 @@
 
 import java.util.ArrayList;
 import java.util.List;
-import java.util.Vector;
 
 import som.vm.Universe;
-import som.vmobjects.SAbstractObject;
-import som.vmobjects.SInvokable;
-import som.vmobjects.SMethod;
-import som.vmobjects.SPrimitive;
-import som.vmobjects.SSymbol;
+import som.vmobjects.*;
```

Comment:
The code uses consistently the explicit imports. Would be good to stick with it.